src/components/ErrorPage/__snapshots__/ErrorPage.test.jsx.snap

```diff
@@ -69,9 +69,13 @@ exports[`<ErrorPage /> renders correctly for 403 errors 1`] = `
             For assistance, please contact the edX Customer Success team at
              
             <a
-              className="default-link standalone-link"
```

Comment:
hmm curious why all these snapshot changes happened?

Reply:
wonder if the snapshot changes are due to something else rather than the text change?

Reply:
Yeah I'm fairly confused as well, trying to see what's up

Reply:
rebase, remove all snapshot changes, then npm run test again?